packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -14,6 +14,15 @@ export default class RunPanelStore {
 
     run_id = '';
 
+    @observable state = {
```

Comment:
Could we rename this to `statistics`, `state` is a bit too generic imo.

Reply:
sure.